### PR TITLE
Add two missing controller authorization checks

### PR DESCRIPTION
**Pull request by jasalisbury**
**Description of change**

Add two missing authorization checks to the API. Any routes added will need an authorization check or need to be explicitly excluded.

**How to test**

* Verify tests pass

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/265

**Checklist**

- [x] Meets issue criteria
- [x] Code tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

File: src/routes/files/handlers.js [hunk 1]
```diff
@@ -4,6 +4,10 @@ import handleErrors from '../../lib/apiErrorHandler';
 import { File } from '../../models';
 import s3Uploader from '../../lib/s3Uploader';
 
+import ActivityReport from '../../policies/activityReport';
```
Comment (dcloud) [inline src/routes/files/handlers.js:7]: I think calling this `ActivityReportPolicy` is a little clearer, as you do elsewhere to avoid name collisions.

**Review by gopar — APPROVED** [review]

File: src/services/activityReports.test.js [hunk 2]
```diff
@@ -61,15 +61,15 @@ describe('Activity Reports DB service', () => {
     });
 
     it('creates a new report with non-grantee recipient', async () => {
-      const beginningARCount = await ActivityReport.count();
       const report = await createOrUpdate({ ...reportObject, activityRecipientType: 'non-grantee' });
-      const endARCount = await ActivityReport.count();
-      expect(endARCount - beginningARCount).toBe(1);
```
Comment (dcloud) [inline src/services/activityReports.test.js:67]: Was this some leftover code (perhaps checking that parallel tests weren't creating extra report records)?

**Review by dcloud — APPROVED** [review]
Looks good to me!

File: src/services/activityReports.test.js (hunk 2 above)
Comment (jasalisbury) [reply src/services/activityReports.test.js:67]: yep!

**Review by rahearn — APPROVED** [review]
> or need to be explicitly excluded.

Is this enforced some way? It's not a requirement of this PR, but I was expecting to see something like pundit's `after_action :verify_authorized` when I read that.

**Comment by jasalisbury** [comment]
> > or need to be explicitly excluded.
> 
> Is this enforced some way? It's not a requirement of this PR, but I was expecting to see something like pundit's `after_action :verify_authorized` when I read that.

No unfortunately it'll have to be a manual review for now. I saw a few `pundit` style libraries for javascript but they were last updated a while ago and not necessary for the pattern so I didn't look at any too long. One of those libraries may have something like `verify_authorized` (or we may be able to come up with something ourselves).

**Pull request merged**